### PR TITLE
Less conversion

### PR DESCRIPTION
**Pull request by mindreeper2420**
Convert `.scss` files to `.less`; add linters for Less and HTML files; update Gulp scripts; update Webpack build scripts.

closes https://github.com/fabric8-ui/fabric8-recommender/issues/314

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -52,7 +52,18 @@ module.exports = function (options) {
     plugins: [
       new DashboardPlugin(),
       extractCSS,
```
Comment (arunkumars08) [inline config/webpack.dev.js]: The declaration for this seems to have been commented. Is this correct?

File: config/webpack.dev.js [hunk 2]
```diff
@@ -52,7 +52,18 @@ module.exports = function (options) {
...
-      extractSASS,
+      // extractSASS,
+      /*
+       * StyleLintPlugin
+       */
+      new StyleLintPlugin({
```
Comment (arunkumars08) [inline config/webpack.dev.js]: Is this defined in this file? I see one in webpack.common.js but there is no reference of that plugin here

File: config/webpack.dev.js (hunk 1 above)
Comment (mindreeper2420) [reply config/webpack.dev.js]: fixed - thanks for the catch!

File: config/webpack.dev.js (hunk 2 above)
Comment (mindreeper2420) [reply config/webpack.dev.js]: Definitely a mistake. I've added the definition to this file.

**Comment by mindreeper2420** [comment]
@arunkumars08 I am unable to run the Webdriver/Selenium process of `npm test` on my Mac for some reason. Before it stalls there, the 4 tests show up as green/passed. Thoughts?

**Comment by mindreeper2420** [comment]
@arunkumars08 with Corinne's help, I was able to get the tests running, but I needed to change a few lines in the test file. I've submitted an issue for it here: https://github.com/fabric8-ui/fabric8-recommender/issues/318

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**